packages/main/src/features/InputSuggestions.js

```diff
@@ -254,23 +255,26 @@ class Suggestions {
 	}
```

Comment:
_getScrollContainer() {
// I guess this also should be await-ed
this._scrollContainer = this._respPopover.getDomRef().shadowRoot.querySelector(".ui5-popover-content");
}